### PR TITLE
2018 DAC CRS codelist updates

### PR DESCRIPTION
**Pull request by andylolz**
DAC codelists were updated in January 2018. Then in June, and again in July.

This PR was originally for January 2018 updates, but I’ve now rolled in June and July too.

NB this PR was generated using IATI/IATI-Codelists-NonEmbedded#172.

### CODE REVIEW

File: xml/SectorCategory.xml [hunk 1]
```diff
@@ -161,8 +161,8 @@
         <codelist-item>
             <code>231</code>
             <name>
-                <narrative>Energy generation, distribution and efficiency – general</narrative>
-                <narrative xml:lang="fr">Énergie : production, distribution et efficacité - général</narrative>
+                <narrative>Energy Policy</narrative>
```
Comment (andylolz) [inline xml/SectorCategory.xml]: This might be a noteworthy change.

**Comment by andylolz** [comment]
This is two months old now, with no response. There’s a new update from DAC waiting. Can someone from @IATI review and merge this, please?

**Comment by samuele-mattiuzzo** [comment]
Hi Andy, thanks for all your reminders and pointers about how old this is between here and Discuss, obviously appreciated! We're sorting a few things out on our side, we'll get to it as soon as time and resources allow. Thanks again :)

**Comment by samuele-mattiuzzo** [comment]
Does #172 need to go in before this? In what capacity are these two issues related?
@andylolz

**Comment by andylolz** [comment]
Many thanks for your response, @samuele-mattiuzzo!

@samuele-mattiuzzo says:
> Does #172 need to go in before this?

This can be treated as independent of #172.

This PR represents updates to some replicated (third party) codelists maintained by the DAC. It’s equivalent to #231.

There’s a [process for management of these codelists](http://reference.iatistandard.org/203/codelists-guides/codelist-management/#updates-to-non-embedded-codelists):

> We aim to make changes to these lists quickly to allow data users to take advantage of the changes immediately. These changes can be made outside of a formal decimal or integer upgrade process.
>
> […]
>
> The IATI Technical Team hold regular team meetings once a week. All outstanding proposals will be discussed at that meeting. The IATI Technical Lead (Bill Anderson) has the ultimate responsibility for decisions taken.

---

@samuele-mattiuzzo says:
> In what capacity are these two issues related?

#172 was used to generate this. It’s a script to fetch these (and other) codelists from managed sources, so that someone (ideally someone who isn’t me) can more easily keep an eye on them and keep them up-to-date.

I’m guessing (but no-one has actually told me this) that #172 is blocked while the secretariat waits to see what’s going on with [the DAC publishing IATI XML](http://www.oecd.org/dac/financing-sustainable-development/development-finance-topics/crs-xml.htm), perhaps with a view to dropping these codelists from the IATI standard? But I’m guessing – I really don’t know.

**Comment by andylolz** [comment]
Closing this, because it seems the intention is to update DAC codelists from the XML source instead.

In that case, I’ve submitted #283.

**Pull request closed without merging**